tensorflow/lite/experimental/micro/examples/micro_speech/apollo3/pushbutton_main.c

```diff
@@ -0,0 +1,339 @@
+/* This file is a modification of the Tensorflow Micro Lite file _main.c
+ * We have retained the original copyright and header information, in
```

Comment:
Can you run clang-format --style=google on this and the other C/C++ source files to get them standardized for layout?